server/routes/overview/overviewController.test.ts

```diff
@@ -92,7 +83,7 @@ describe('overviewController', () => {
       prisonerSummary: expectedPrisonerSummary,
       tab: expectedTab,
       prisonNumber,
-      goals,
+      goals: res.locals.goals.goals,
```

Comment:
I think this line, and 130, should be:
```
goals: res.locals.goals,
```
IE. we want the whole object inc. the problemRetrievingData field, not just the array of goals.